docs/developer/decision-records/2022-07-01-get-claims/apply-policies-flow.puml

```diff
@@ -0,0 +1,35 @@
+@startuml
+autonumber "<b>0'.'"
+box "Participant A" #LightPink
+Participant "Participant A\nClient" as ParticipantA
+Participant "Participant A\nDID Document" as DID
+Participant "Participant A\nIdentityHub" as IDHUB
+end box
+box "Participant B" #LightGreen
+Participant "Participant B\nAPI" as B
+end box
+box "Credential issuer" #LightBlue
+Participant "Credential issuer DID" as ISSUER
+end box
+ParticipantA -> B ++: Request:\n - DID JWS
+activate ParticipantA
+group authorize request
+  B -> DID ++: Get DID Document (contains IdentityHub URL)
+  return DID Document
+  B -> B: Verify JWS
+  group Get Claims
+  B -> B: Extract IdentityHub URL
+  B -> IDHUB ++: Get credentials
+  return List of JWS
+  loop for each JWS
+  B -> B: Get issuer DID
```

Comment:
nit: 
```suggestion
  B -> B: Extract issuer DID
```
To be consistent with arrow 11. that `extract` is internal operation and `get` would mean getting someting from another service/component